js/libs/builder.js

```diff
@@ -1035,6 +1035,9 @@ new Vue({
       }
     }, savedLinkData);
 
+    this.toggleAccessType('insert', this.showExtraAdd);
```

Comment:
```js
      this.toggleAccessType('insert', this.showExtraAdd);
      this.toggleAccessType('update', this.showExtraEdit);
```

This is called twice in this file. I think they should be collated into a `this.updateAccessTypes()` function. This way when we have new access rules to consider, there's only one place to update.

Reply:
@tonytlwu, i create the updateAccessTypes function and moved toggling of rules, there